firebase-storage/src/androidMain/kotlin/dev/gitlive/firebase/storage/storage.kt

```diff
@@ -88,6 +91,12 @@ actual class StorageReference(val android: com.google.firebase.storage.StorageRe
     }
 }
 
+actual class ListResult(val android: com.google.firebase.storage.ListResult) {
+    actual val prefixes: List<StorageReference> get() = android.prefixes.map { StorageReference(it) }
+    actual val items: List<StorageReference> get() = android.items.map { StorageReference(it) }
+    actual val pageToken: String? get() = android.pageToken
```

Comment:
using getters here but standard initialization in ios - if these properties are immutable (im assuming they are) then by lazy { .. } would be best otherwise if they are mutable then should be getters

Reply:
Went with the same standard initialization as the other platforms. These properties are, in-fact, immutable.